l10n_it_riba_oca/hooks.py

```diff
@@ -0,0 +1,94 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+OLD_MODULE_NAME = "l10n_it_riba"
+NEW_MODULE_NAME = "l10n_it_riba_oca"
```

Comment:
```suggestion
OLD_MODULE_NAME = "l10n_it_ricevute_bancarie"
NEW_MODULE_NAME = "l10n_it_riba_oca"
```
Questo hook serve per chi migra da `14.0`, dove il modulo si chiama `l10n_it_ricevute_bancarie`, alla `16.0` dove il modulo, dopo questa PR, si chiamerà `l10n_it_riba_oca`.

Quindi anche qui va solo fatta la sostituzione `l10n_it_riba` -> `l10n_it_riba_oca`.

Per la migrazione dati ti sto proponendo io una PR come abbiamo detto poco fa in chiamata.